crates/oxc_linter/src/snapshots/checked_requires_onchange_or_readonly.snap

```diff
@@ -0,0 +1,101 @@
+   · ────────────────────────────────────────────────
+   ╰────
+  help: Remove either `checked` or `defaultChecked`.
+
+  ⚠ eslint-plugin-react(checked-requires-onchange-or-readonly): `checked` should be used with either `onChange` or `readOnly`.
+   ╭─[checked_requires_onchange_or_readonly.tsx:1:1]
+ 1 │ <input type='checkbox' checked defaultChecked />
+   · ────────────────────────────────────────────────
+   ╰────
+  help: Add either `onChange` or `readOnly`.
+
+  ⚠ eslint-plugin-react(checked-requires-onchange-or-readonly): `checked` should be used with either `onChange` or `readOnly`.
+   ╭─[checked_requires_onchange_or_readonly.tsx:1:1]
+ 1 │ React.createElement('input', { checked: false })
+   · ────────────────────────────────────────────────
+   ╰────
+  help: Add either `onChange` or `readOnly`.
+
+  ⚠ eslint-plugin-react(checked-requires-onchange-or-readonly): Use either `checked` or `defaultChecked`, but not both.
+   ╭─[checked_requires_onchange_or_readonly.tsx:1:1]
+ 1 │ React.createElement('input', { checked: true, defaultChecked: true })
+   · ─────────────────────────────────────────────────────────────────────
+   ╰────
+  help: Remove either `checked` or `defaultChecked`.
+
+  ⚠ eslint-plugin-react(checked-requires-onchange-or-readonly): `checked` should be used with either `onChange` or `readOnly`.
+   ╭─[checked_requires_onchange_or_readonly.tsx:1:1]
+ 1 │ React.createElement('input', { checked: true, defaultChecked: true })
+   · ─────────────────────────────────────────────────────────────────────
+   ╰────
+  help: Add either `onChange` or `readOnly`.
+
+  ⚠ eslint-plugin-react(checked-requires-onchange-or-readonly): Use either `checked` or `defaultChecked`, but not both.
+   ╭─[checked_requires_onchange_or_readonly.tsx:1:1]
+ 1 │ <input type='checkbox' checked defaultChecked />
+   · ────────────────────────────────────────────────
+   ╰────
+  help: Remove either `checked` or `defaultChecked`.
+
+  ⚠ eslint-plugin-react(checked-requires-onchange-or-readonly): `checked` should be used with either `onChange` or `readOnly`.
+   ╭─[checked_requires_onchange_or_readonly.tsx:1:1]
+ 1 │ <input type='checkbox' checked defaultChecked />
+   · ────────────────────────────────────────────────
+   ╰────
+  help: Add either `onChange` or `readOnly`.
+
+  ⚠ eslint-plugin-react(checked-requires-onchange-or-readonly): Use either `checked` or `defaultChecked`, but not both.
+   ╭─[checked_requires_onchange_or_readonly.tsx:1:1]
+ 1 │ <input type='checkbox' checked defaultChecked />
+   · ────────────────────────────────────────────────
+   ╰────
+  help: Remove either `checked` or `defaultChecked`.
+
+  ⚠ eslint-plugin-react(checked-requires-onchange-or-readonly): `checked` should be used with either `onChange` or `readOnly`.
+   ╭─[checked_requires_onchange_or_readonly.tsx:1:1]
+ 1 │ <input type='checkbox' checked defaultChecked />
+   · ────────────────────────────────────────────────
```

Comment:
Let's point the span to the `checked` attribute

Reply:
Thank you!
I have corrected it to point to the `checked` attribute.
e8e9bb3509261b48ff243eb0c1f23fc5036d9964